src/Microsoft.DocAsCode.Dfm.VscPreview/Microsoft.DocAsCode.Dfm.VscPreview.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\Shared\base.props" />
+  <PropertyGroup>
+    <ProjectGuid>{6E31517C-B140-413F-B854-F72E25F4DAB2}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="System.Collections.Immutable, Version=1.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Oh,I forgot to check it!

---
In reply to: [73995641](https://github.com/dotnet/docfx/pull/583#discussion_r73995641) [](ancestors = 73995641)